<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExitEditWindow</class>
 <widget class="QDialog" name="ExitEditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>138</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Edit Exit</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/images/icons/kale-doors-new.png</normaloff>:/images/icons/kale-doors-new.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="4">
    <widget class="QLabel" name="label_X">
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_Level">
     <property name="text">
      <string>Level</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_Screen">
     <property name="text">
      <string>Screen</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="8">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QSpinBox" name="spinBox_Y">
     <property name="maximum">
      <number>11</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3">
    <widget class="QSpinBox" name="spinBox_Screen">
     <property name="maximum">
      <number>15</number>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QLabel" name="label_Y">
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QSpinBox" name="spinBox_X">
     <property name="maximum">
      <number>15</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="7">
    <widget class="QComboBox" name="comboBox_Type"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_Type">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="label_BossStuff">
     <property name="text">
      <string>Boss Room (levels 000-007)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLabel" name="label_BossX">
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_BossLevel">
     <property name="text">
      <string>Level</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_BossScreen">
     <property name="text">
      <string>Screen</string>
     </property>
    </widget>
   </item>
   <item row="3" column="7">
    <widget class="QSpinBox" name="spinBox_BossY">
     <property name="maximum">
      <number>11</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="horizontalSpacer2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3">
    <widget class="QSpinBox" name="spinBox_BossScreen">
     <property name="maximum">
      <number>15</number>
     </property>
    </widget>
   </item>
   <item row="3" column="6">
    <widget class="QLabel" name="label_BossY">
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QSpinBox" name="spinBox_BossX">
     <property name="maximum">
      <number>15</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spinBox_Screen</tabstop>
  <tabstop>spinBox_X</tabstop>
  <tabstop>spinBox_Y</tabstop>
  <tabstop>comboBox_Type</tabstop>
  <tabstop>spinBox_BossScreen</tabstop>
  <tabstop>spinBox_BossX</tabstop>
  <tabstop>spinBox_BossY</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExitEditWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExitEditWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
